share/extender/build_input.yml

```diff
@@ -9,10 +9,15 @@ context:
                   "libengine.lib","libengine_release.lib","libprofile.lib","libprofile_null.lib","libremotery.lib","libremotery_null.lib","libprofilerext.lib","libprofilerext_null.lib","libcrashext","libcrashext_null","librecord.lib","librecord_null.lib","libgameobject.lib","libddf.lib","libresource.lib","libgamesys.lib","libgamesys_model.lib","libgamesys_rig.lib","librig.lib","libgamesys_model_null.lib","libgamesys_rig_null.lib","librig_null.lib","libgraphics.lib","libgraphics_null.lib","libgraphics_transcoder_null.lib","libgraphics_transcoder_basisu.lib","libbasis_transcoder.lib","libphysics_2d.lib","libphysics_2d_defold.lib","libphysics_3d.lib","libphysics_null.lib","libBulletDynamics.lib","libBulletCollision.lib","libplatform.lib","libplatform_null.lib","libLinearMath.lib","libbox2d.lib","librender.lib","librender_font_default.lib",
                   "libscript.lib","libluajit.lib-5.1","libextension.lib","libhid.lib","libhid_null.lib","libinput.lib","libparticle.lib","librig.lib","libdlib.lib","libimage.lib","libimage_null.lib","libdmglfw.lib","libgui.lib","libcrashext.lib","libsound.lib","libsound_null.lib","libtremolo.lib","libvpx.lib","libliveupdate.lib",
                   "engine_service","engine_service_null","libengine_service.lib","libengine_service_null.lib",
-                  "script_box2d", "script_box2d_defold", "script_box2d_null", "libscript_box2d.lib", "libscript_box2d_defold.lib", "libscript_box2d_null.lib"]
-    allowedSymbols: ["DefaultSoundDevice", "NullSoundDevice", "AudioDecoderWav", "CrashExt", "AudioDecoderStbVorbis", "GraphicsAdapterWebGPU",
+                  "script_box2d", "script_box2d_defold", "script_box2d_null", "libscript_box2d.lib", "libscript_box2d_defold.lib", "libscript_box2d_null.lib",
+                  "opus", "libopus.lib",
+                  "decoder_wav", "libdecoder_wav.lib",
+                  "decoder_ogg", "libdecoder_ogg.lib",
+                  "decoder_opus", "libdecoder_opus.lib"]
+    allowedSymbols: ["DefaultSoundDevice", "NullSoundDevice", "AudioDecoderWav", "CrashExt", "AudioDecoderStbVorbis", "AudioDecoderOpus", "GraphicsAdapterWebGPU",
                     "ProfilerExt", "GraphicsAdapterOpenGL", "GraphicsAdapterOpenGLES", "GraphicsAdapterVulkan", "GraphicsAdapterNull",
-                    "LiveUpdateExt", "ScriptBox2DExt", "ScriptImageExt", "ScriptModelExt", "ScriptTypesExt"]
+                    "LiveUpdateExt", "ScriptBox2DExt", "ScriptImageExt", "ScriptModelExt", "ScriptTypesExt",
+                    "ResourceTypeOgg", "ResourceTypeWav", "ResourceTypeOpus"]
```

Comment:
I think we should add `"ResourceTypeOgg", "ResourceTypeWav"` to symbols section for every platform because won't be included.

Reply:
I don't understand? "because won't be included" ?

Reply:
Sorry, my bad. I missed that "ResourceTypeOgg", "ResourceTypeWav" have already been defined in top-level `symbols`